public/css/style.css

```diff
@@ -1,694 +1,719 @@
-  font-size: 2rem; }
+.topnav #myLinks a:hover{
+background-color: #f8f9fa;
+}
 
 .navbar-nav {
-  display: flex;
-  align-items: center; 
-  flex-direction: row; }
-
-  .navbar li {
-    list-style: none;
-  }
-
-  /* About */
-.about-jumbo.jumbotron {
-  background: #ABBDF0 url("/images/couple-cooking.jpg") no-repeat center;
-  background-size: cover;
-}
-.about-jumbo p {
-  color: #000;
-}
-.about-jumbo .searchbox {
-  position: relative;
-  background: none;
-  z-index: 2;
-}
-.about-jumbo .searchbox::before {
-  content: '';
-  position: absolute;
-  left: 0;
-  top: 0;
-  width: 100%;
-  height: 100%;
-  background: #fff;
-  opacity: .8;
-  z-index: -1;
+	display: flex;
+	align-items: center;
+	flex-direction: row;
 }
 
-/* Hide the links inside the navigation menu (except for logo/home) */
-.topnav #myLinks {
-  display: none; }
-
-/* MENU links */
-.topnav #myLinks a{
-  color: #061404;
-  padding:14px 20px;
-  text-decoration: none;
-  font-size: 1rem;
-  display: flex;
-  align-items: center; 
+.navbar li {
+	list-style: none;
 }
 
 .topnav a span {
-  padding-right: 15px; 
+	padding-right: 15px;
 }
 
-/* HAMBURGUER menu */
+/* HAMBURGUER ICON */
 .topnav a.icon {
-  color: #061404; 
+	color: #061404;
 }
 
 /* ACTIVE link (or home/logo) */
 .active {
-  display: block; }
-
-.signUp {
-  position: initial;
-  display: block; }
+	display: block;
+}
 
-/* Buttons */
+/* ----BUTTONS----- */
 .btn {
-  background: #ff914d;
-  padding: 10px 24px;
-  color: #fff;
-  text-align: center; }
-  
+	background: #ff914d;
+	padding: 10px 24px;
+	color: #fff;
+	text-align: center;
+}
+
 .btn:hover {
-  padding: 10px 24px;
-  border: 1px solid #ff914d;
-  background-color: #fff;
-  color: #061404; }
+	padding: 10px 24px;
+	border: 1px solid #ff914d;
+	background-color: #fff;
+	color: #061404;
+}
 
 .btn-search-home {
-    background: #218838;
+	background: #218838;
 }
-    
+
 .btn-search-home:hover {
-    border: 1px solid #218838;
-    background-color: #fff;
-    color: #061404; }
+	border: 1px solid #218838;
+	background-color: #fff;
+	color: #061404;
+}
 
-/*Save recipe button*/
-.saveRecipeBtn {
-  border: 0!important;
-  background: transparent;
-  outline: 0!important;
-  transition: all .2s ease-in-out;
-  
+.signUp {
+	position: initial;
+	display: block;
 }
-.saveRecipeBtn:hover {
-  transform: scale(1.3);
+
+/* Account Button  */
+.dropdown {
+	display: inline-block;
+}
+.dropdown-item {
+	cursor: pointer;
+}
+.dropdown-item button {
+	border: none;
+}
+.dropdown-item:link {
+	text-decoration: none;
+	background-color: transparent;
+}
+.dropdown-item:hover {
+	color: #061404;
+	text-decoration: none;
+	background-color: #f8f9fa;
+}
+.dropdown-item:active {
+	color: white;
+	text-decoration: none;
+	background-color: #218838;
+}
+.dropdown-item:visited {
+	color: black;
+	text-decoration: none;
 }
 
-/*google btn*/
-.modal-body .btn-google {
-  width: 100%;
-  display: block;
-  text-align: center;
+#padding{
```

Comment:
Is it necessary to add an id property for padding?
If not, you can completely remove this # property and just add padding to the necessary element.
Otherwise, keep the same

Reply:
IDs removed